05_astroblasto doesn't build

The [05_astroblasto](https://github.com/ggez/ggez/blob/master/examples/05_astroblasto.rs) example doesn't build.

`nalgebra::Point2<f32>` does not implement `Into<mint::Point2<f32>>`.

**To Reproduce**
 * create a new Rust binary project
 * add `ggez`, `rand` and `nalgebra` to `Cargo.toml`
 * copy-paste the example to `src/main.rs`
 * run `cargo build`

**Screenshots or pasted code**

```rust
fn draw_actor(
  assets: &mut Assets,
  ctx: &mut Context,
  actor: &Actor,
  world_coords: (f32, f32),
) -> GameResult {
  let (screen_w, screen_h) = world_coords;
  let pos = world_to_screen_coords(screen_w, screen_h, actor.pos);
  let image = assets.actor_image(actor);
  let drawparams = graphics::DrawParam::new()
    .dest(pos)
    .rotation(actor.facing as f32)
    .offset(Point2::new(0.5, 0.5));
  graphics::draw(ctx, image, drawparams)
}
```

```
error[E0277]: the trait bound `mint::vector::Point2<f32>: std::convert::From<nalgebra::geometry::point::Point<f32, nalgebra::base::dimension::U2>>` is not satisfied
   --> src\main.rs:415:6
    |
415 |     .dest(pos)
    |      ^^^^ the trait `std::convert::From<nalgebra::geometry::point::Point<f32, nalgebra::base::dimension::U2>>` is not implemented for `mint::vector::Point2<f32>`
    |
    = help: the following implementations were found:
              <mint::vector::Point2<T> as std::convert::From<[T; 2]>>
              <mint::vector::Point2<T> as std::convert::From<mint::vector::Vector2<T>>>
    = note: required because of the requirements on the impl of `std::convert::Into<mint::vector::Point2<f32>>` for `nalgebra::geometry::point::Point<f32, nalgebra::base::dimension::U2>`
```

**Hardware and Software:**
 - ggez version: release `0.5.1`
 - OS: Windows 10

--- a/examples/05_astroblasto.rs
+++ b/examples/05_astroblasto.rs
@@ -10,7 +10,7 @@ use ggez::event::{self, EventHandler, KeyCode, KeyMods};
 use ggez::graphics;
 use ggez::timer;
 use ggez::{Context, ContextBuilder, GameResult};
-use nalgebra as na;
+use ggez::nalgebra as na;
 use rand;
 
 use std::env;

--- a/examples/bunnymark.rs
+++ b/examples/bunnymark.rs
@@ -4,7 +4,7 @@
 use std::env;
 use std::path;
 
-use nalgebra as na;
+use ggez::nalgebra as na;
 use rand::rngs::ThreadRng;
 use rand::{self, Rng};
 

--- a/examples/cube.rs
+++ b/examples/cube.rs
@@ -17,7 +17,7 @@ use gfx::Factory;
 use ggez::event;
 use ggez::graphics;
 use ggez::{Context, GameResult};
-use nalgebra as na;
+use ggez::nalgebra as na;
 use std::env;
 use std::f32;
 use std::path;

--- a/examples/graphics_settings.rs
+++ b/examples/graphics_settings.rs
@@ -17,7 +17,7 @@ use structopt::StructOpt;
 use std::env;
 use std::path;
 
-use nalgebra as na;
+use ggez::nalgebra as na;
 type Point2 = na::Point2<f32>;
 
 struct WindowSettings {

--- a/examples/hello_canvas.rs
+++ b/examples/hello_canvas.rs
@@ -7,7 +7,7 @@ use nalgebra;
 use ggez::event;
 use ggez::graphics::{self, Color};
 use ggez::{Context, GameResult};
-use nalgebra as na;
+use ggez::nalgebra as na;
 use std::env;
 use std::path;
 

--- a/examples/transforms.rs
+++ b/examples/transforms.rs
@@ -5,7 +5,7 @@ use nalgebra;
 use ggez::event::{self, KeyCode, KeyMods};
 use ggez::graphics::{self, DrawMode};
 use ggez::{Context, GameResult};
-use nalgebra as na;
+use ggez::nalgebra as na;
 use std::env;
 use std::path;
 

